terraform/modules/jvs-services/variables.tf

```diff
@@ -0,0 +1,213 @@
+    developers = list(string)
+    invokers   = list(string)
+  })
+  default = {
+    admins     = []
+    developers = []
+    invokers   = []
+  }
+}
+
+variable "jvs_cert_rotator_service_name" {
+  description = "Name for JVS cert rotator service."
+  type        = string
+  default     = "jvs-cert-rotator"
+}
+
+variable "jvs_cert_rotator_service_account" {
+  description = "Service account for JVS cert rotator service."
+  type        = string
+}
+
+variable "jvs_cert_rotator_service_image" {
+  description = "Container image for JVS cert rotator service."
+  type        = string
+  default     = "gcr.io/cloudrun/hello:latest"
+}
+
+variable "jvs_cert_rotator_service_iam" {
+  description = "IAM member bindings for JVS cert rotator service."
+  type = object({
+    admins     = list(string)
+    developers = list(string)
+    invokers   = list(string)
+  })
+  default = {
+    admins     = []
+    developers = []
+    invokers   = []
+  }
+}
+
+variable "jvs_public_key_service_name" {
+  description = "Name for JVS public key service."
+  type        = string
+  default     = "jvs-public-key"
+}
+
+variable "jvs_public_key_service_account" {
+  description = "Service account for JVS public key service."
+  type        = string
+}
+
+variable "jvs_public_key_service_image" {
+  description = "Container image for JVS public key service."
+  type        = string
+  default     = "gcr.io/cloudrun/hello:latest"
+}
+
+variable "jvs_public_key_service_iam" {
+  description = "IAM member bindings for JVS public key service."
+  type = object({
+    admins     = list(string)
+    developers = list(string)
+    invokers   = list(string)
+  })
+  default = {
+    admins     = []
+    developers = []
+    # Public key service is meant to be public.
+    invokers = ["allUsers"]
+  }
+}
+
+variable "keyring_id" {
+  description = "KMS keyring to create signing keys."
+  type        = string
+}
+
+variable "key_name" {
+  description = "KMS key id for use with signing."
+  type        = string
+}
+
+variable "key_ttl" {
+  description = "the length of time that we expect a key to be valid for."
+  default     = "10m"
+  type        = string
+}
+
+variable "key_grace_period" {
+  description = "length of time between when we rotate the key and when an old Key Version is no longer valid and available."
+  type        = string
+  default     = "5m"
+}
+
+variable "key_disabled_period" {
+  description = "length of time between when the key is disabled and when we delete the key."
+  type        = string
+  default     = "5m"
+}
+
+variable "key_propagation_delay" {
+  description = "length of time that it takes for a change in the key in KMS to be reflected in the client."
+  type        = string
+  default     = "2m"
+}
+
+variable "key_rotation_cadence" {
+  type        = number
+  default     = 5
+  description = "Cadence (expressed in minutes) to run the certificate rotator on. If set to 0, key rotation won't be scheduled."
+}
```

Comment:
Per the evolving Terraform style guide, this should be "key_rotation_minutes": [go/bets-terraform-guide](https://goto.google.com/bets-terraform-guide)